### DISCUSSION TITLE
Can I change to use another ckpt? because it seems that JuggernautXLv7.safetensors is a default setting?

### DISCUSSION THREAD

**Discussion by jjiikkkk** [lllyasviel/Fooocus#2741] [Q&A] [answered]
when starting 
!python entry_with_update.py

this ckpt is downloaded automatically, thanks.

**Comment by ILikeToasters** [comment, marked as answer]
Yes you can download other checkpoints that are SDXL and put in your checkpoint folder. By default Fooocus will download if you don't based on what is in the preset files which can be edited. You will find a models folder in your Fooocus folder and in there you will find the checkpoint folder. You can select between checkpoints in the drop down in the interface under models.

**Reply by mashb1t** [reply]
in addition to the information above you can also copy an existing preset and create your own, calling it in a separate .bat file on startup or using it directly with the argument `--preset xyz`. 
Happy prompting!
